Tests/Layout/RestrictedBrickSizeTest.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  RestrictedBrickSizeTest.swift
+//  BrickKit
+//
+//  Created by Peter Cheung on 7/28/17.
+//  Copyright © 2017 Wayfair. All rights reserved.
+//
+
+import XCTest
+@testable import BrickKit
+
+class RestrictedBrickSizeTest: XCTestCase {
+    var brickCollectionView: BrickCollectionView!
+    
+    override func setUp() {
+        super.setUp()
+        
+        brickCollectionView = BrickCollectionView(frame: CGRect(x: 0, y: 0, width: 320, height: 480))
+    }
+    
+    func testChangeBrickSizeGreaterThanMinimum() {
+        
+        let section = BrickSection("Test Section", bricks: [
+            DummyResizableBrick("Brick", height: .restricted(restriction: RestrictedBrickSize(size: .auto(estimate: .fixed(size: 100)), restrictedSize: .minimumSize(size: .fixed(size: 100)))), setHeight: 200)
+            ])
+        
+        brickCollectionView.setupSectionAndLayout(section)
+        
+        let cell = brickCollectionView.cellForItem(at: IndexPath(item: 0, section: 1)) as? DummyResizableBrickCell
```

Comment:
Check the indexpath by calling `brickCollectionView.indexPathsForBricksWithIdentifier`. It's a safer way to get to the right cell